userdocs/contents/vmware.rst

```diff
@@ -0,0 +1,34 @@
+============================
+Using Warewulf on VMWare
```

Comment:
I like the idea of including this; but it needs to be expanded. Should provide more of a walkthrough for how to set this up.